aimmo-game/simulation/communicator.py

```diff
@@ -0,0 +1,15 @@
+import requests
+
+class Communicator():
```

Comment:
I feel like the purpose of this class isn't 100% clear, what is it communicating with? Maybe improve its name or add a docstring